src/Components/Home.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+export default function Home() {
+  return (
+    <div style={{
+      width: '75%', margin: 'auto', textAlign: 'center', fontSize: 19, marginTop: '5%',
+    }}
+    >
+      <h1>Welcome to our page</h1>
+      <p style={{ textAlign: 'start' }}>
+        Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+        Lorem Ipsum has been the industry&lsquo s standard dummy text ever since the 1500s,
+        when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+        It has survived not only five centuries, but also the leap into electronic typesetting,
+        remaining essentially unchanged. It was popularised in the 1960s
+        with the release of Letraset
+        sheets
+        containing Lorem Ipsum passages, and more recently with
+        desktop publishing software like Aldus
+        PageMaker including versions of Lorem Ipsum.
+      </p>
+      <p style={{ textAlign: 'start' }}>
+        It is a long established fact that a reader will be distracted by
+        the readable content of
+        a page when looking at its layout. The point of using
+        Lorem Ipsum is that it has a more-or-less
+        normal distribution of letters, as opposed to using
+        Content here, content here, making it look
+        like readable English. Many desktop publishing packages
+        and web page editors now use Lorem Ipsum
+        as their default model text, and a search for
+        lorem ipsum will uncover many web sites still in
+        their infancy. Various versions have evolved over
+        the years, sometimes by accident, sometimes on
+        purpose (injected humour and the like).
+      </p>
+    </div>
+  );
+}
```

Comment:
- [ ] I had this issue while trying to run the project locally, so to fix it you have to create `.env file and paste SKIP_PREFLIGHT_CHECK=true`

![user-13](https://user-images.githubusercontent.com/55635977/140175925-a5ab9df5-a6e3-45ff-be70-4775e30258dd.png)